tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -529,5 +529,6 @@
   "trace.wordpress_callbacks": "trace_wordpress_callbacks",
   "trace.wordpress_enhanced_integration": "trace_wordpress_enhanced_integration",
   "trace.wordpress_additional_actions": "trace_wordpress_additional_actions",
-  "trace.sidecar_trace_sender": "trace_sidecar_trace_sender"
+  "trace.sidecar_trace_sender": "trace_sidecar_trace_sender",
+  "DD_TRACE_SAMPLING_RULES_FORMAT": "trace_sample_rules_format"
```

Comment:
nit: I suggest putting this next to the other `DD_TRACE_SAMPLING_*` config so it's easiest to spot but doesn't matter! 

Don't forget to raise a PR against dd-go that makes this change too 🙂

Reply:
This change is done, and Pierre already added this to the other repo while he was adding other unrelated mappings.